docs/ide/creating-solutions-and-projects.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: Work with solutions and projects
+title: "Create &amp; use Visual Studio projects &amp; solutions"
```

Comment:
@ghogen I can't find supporting documentation, but I believe that the HTML in the metadata title will display as literal text in Google search results. I spotted this once a few years ago in another content set. Not 100% sure, but if you publish, you might check on it in a search result that will return this article. 👍🏻

Reply:
```suggestion
title: "Create and use Visual Studio projects and solutions"
```

Reply:
@j-martens Can we just use "and" here?